src/components/modals/FlipInputModal.js

```diff
@@ -106,8 +109,9 @@ class FlipInputModalComponent extends React.PureComponent<Props, State> {
   }
 
   renderExchangeRates = () => {
-    const { primaryInfo, secondaryInfo, fiatPerCrypto, errorMessage, theme } = this.props
+    const { primaryInfo, secondaryInfo, fiatPerCrypto, theme } = this.props
     const styles = getStyles(theme)
+    const errorMessage = this.state.errorMessage ?? this.props.errorMessage
     return (
       <View style={styles.exchangeRateContainer}>
         {errorMessage ? (
```

Comment:
We agreed that if you work on code that has a "fuzy-boolean" like this: `errorMessage ?` you change it to an implicit check like this: `errorMessage != null` or whatever is needed.